aspnetcore/signalr/configuration.md

```diff
@@ -14,15 +14,16 @@ uid: signalr/configuration
 
 ASP.NET Core SignalR supports two protocols for encoding messages: [JSON](https://www.json.org/) and [MessagePack](https://msgpack.org/index.html). Each protocol has serialization configuration options.
 
-JSON serialization can be configured on the server using the [AddJsonProtocol](/dotnet/api/microsoft.extensions.dependencyinjection.jsonprotocoldependencyinjectionextensions.addjsonprotocol) extension method, which can be added after [AddSignalR](/dotnet/api/microsoft.extensions.dependencyinjection.signalrdependencyinjectionextensions.addsignalr) in your `Startup.ConfigureServices` method. The `AddJsonProtocol` method takes a delegate that receives an `options` object. The [PayloadSerializerSettings](/dotnet/api/microsoft.aspnetcore.signalr.jsonhubprotocoloptions.payloadserializersettings) property on that object is a JSON.NET `JsonSerializerSettings` object that can be used to configure serialization of arguments and return values. See the [JSON.NET Documentation](https://www.newtonsoft.com/json/help/html/Introduction.htm) for more details.
+::: moniker range=">= aspnetcore-3.0"
```

Comment:
I wonder if we should be mentioning the `AddNewtonsoftJsonProtocol` extension method in this doc

Reply:
Yes it does

Reply:
The 2.2-to-3.0 migration doc discusses AddNewtonsoftJsonProtocol, but not this SignalR configuration doc. Should we copy the [Switch to Newtonsoft.Json](https://github.com/aspnet/AspNetCore.Docs/blob/f849554c6000e74e9376d359da20c686673ab880/aspnetcore/migration/22-to-30.md#switch-to-newtonsoftjson) section from the migration doc to this doc?

Reply:
Yeah, it looks pretty good